src/LightStep/LightStepSpanRecorder.cs

```diff
@@ -62,5 +62,13 @@ public IEnumerable<SpanData> GetSpans()
                 return Spans.ToList();
             }
         }
+
+        public int GetSpanCount()
+        {
+            lock (Spans)
```

Comment:
BTW, does the benefit brought by locking pays for the cost for read here?

Reply:
After some discussion, the lock isn't terribly necessary here.